tests/features/collection/collection_overview.feature

```diff
@@ -2,17 +2,13 @@
 Feature: Collections Overview
 
   Scenario: Check visibility of "Collections" menu link.
-    Given I am an anonymous user
-    Then I should see the link "Collections"
-    When I click "Collections"
-    Then I should see the heading "Collections"
-    # Check that all logged in users can see and access the link as well.
+    # Check that all logged in users can see and access the link.
```

Comment:
We had a huge discussion on this yesterday. What was the conclusion, is that these tests are there for a reason. The initial idea is that the "Collections" should appear for everyone. After yesterday's discussion, the idea is that we do not tamper with the tests in order for the site to work. If we were to have the "Collections" option disappear for the anonymous users, we would need a technical ticket specifically for this. Now, it seems like this is altered just because the theme is in place. 
This should be rejected from the wireframes, or have a ticket specifically to remove it.

The same applies for "Sign in". This should not change as part of a theme implementation. This isn't your job and everything should be ready for you. So we need to create tickets for these two after the client approves this. Tests are a bit sacred.!